### PR TITLE
Token selector keyboard navigation

### PR DESCRIPTION
**Pull request by ted-palmer**

### CODE REVIEW

File: packages/ui/src/components/primitives/AccessibleList.tsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+}
+
+type ListItemProps = {
+  children: React.ReactNode
+  value: string
+  css?: Styles
+  asChild?: boolean
+}
+
+export const ListItem = forwardRef<HTMLButtonElement, ListItemProps>(
```
Comment (pedromcunha) [inline packages/ui/src/components/primitives/AccessibleList.tsx:60]: I feel like this should be named: AccessibleListItem, so the hierarchy is clear

File: packages/ui/src/components/primitives/Input.tsx [hunk 2]
```diff
@@ -53,66 +54,64 @@ type StyledInputCssVariants = NonNullable<
...
-const Input: FC<
+const Input = forwardRef<
+  HTMLInputElement,
   Omit<HTMLProps<HTMLInputElement>, 'size'> &
     PropsWithChildren & {
       icon?: ReactNode
       iconPosition?: 'left' | 'right'
       iconCss?: Styles
       containerCss?: Styles
-      inputRef?: (element: HTMLInputElement) => void
```
Comment (pedromcunha) [inline packages/ui/src/components/primitives/Input.tsx:63]: I see we removed inputRef, was it not used anywhere else?

**Review by pedromcunha — CHANGES_REQUESTED** [review]

File: packages/ui/src/components/primitives/AccessibleList.tsx (hunk 1 above)
Comment (ted-palmer) [reply packages/ui/src/components/primitives/AccessibleList.tsx:60]: Will do!

File: packages/ui/src/components/primitives/Input.tsx (hunk 2 above)
Comment (ted-palmer) [reply packages/ui/src/components/primitives/Input.tsx:63]: Updated it to pass the ref directly into the component like `<Input ref={ref} `

**Pull request merged**